DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -139,6 +140,11 @@ class TTTrack : public TTTrack_TrackWord {
   unsigned int etaSector() const { return theEtaSector_; }
   void setEtaSector(unsigned int aSector) { theEtaSector_ = aSector; }
 
+  /// GTT Link Information
+  unsigned int GTTLinkID() const { return (eta() >= 0 ? 1 : 0) + (2 * phiSector()); }
+  int GTTLinkIndex() const { return theGTTLinkIndex_; }
```

Comment:
Yes, I'd say these should be
```suggestion
  unsigned int gttLinkID() const { return (eta() >= 0 ? 1 : 0) + (2 * phiSector()); }
  int gttLinkIndex() const { return theGTTLinkIndex_; }
```
as you mention on Mattermost